### ISSUE TITLE
Extract PTM descriptions from Unimod

### ISSUE THREAD

**Issue by wsnoble** [freejstone/CONGA#24]
Bo pointed out that we could extract candidate PTM identifications from Unimod based on the observed delta mass. This would make CONGA much more useful to the end user.

**Comment by mriffle** [comment]
Not saying you shouldn't do this, but Limelight will basically do this for you--it links out to Unimod for open mod masses (rounded to some scale). Part of the challenge w/ unimod is that, depending on the scale of your rounding, you will match different mods. And with open mod masses, since they include the error from the ID (entire delta mass being considered a mod), you won't ever exactly match anything in unimod.

**Comment by freejstone** [comment]
Yes I had wondered if Limelight did this already anyhow, but have been meaning to ask you about that Mike. Thanks!
